docs/src/man1/cdist.rst

```diff
@@ -153,9 +153,10 @@ Configure/install one or more hosts.
 
 .. option:: -j [JOBS], --jobs [JOBS]
 
-    Specify the maximum number of parallel jobs. Global
-    explorers, object prepare and object run are supported
-    (currently in beta).
+    Operate in parallel in specified maximum number of
+    jobs. Global explorers, object prepare and object run
+    are supported. Without argument CPU count is used by
+    default. Currently in beta.
```

Comment:
This is for -j option which specifies maximum jobs for global explorers, object prepare and object run within one host.

-p is not in beta.

So we now have -p HOST_MAX and for each host process -j JOBS parallel jobs.